windshaft/driver.js

```diff
@@ -10,10 +10,11 @@ var uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-
 // NB: interactivity only works with string columns
 var baseRecordQuery = ["(SELECT geom, uuid::varchar, data::varchar, occurred_from::varchar, ",
                        "occurred_to::varchar ",
-                       "FROM ashlar_record"
+                       "FROM ashlar_record WHERE schema_id IN ",
+                       "(SELECT uuid FROM ashlar_recordschema "
                        ].join("");
-var filterRecordQuery = " AND record_type_id = '";
-var endRecordQuery = ") AS ashlar_record";
+var filterRecordQuery = " WHERE record_type_id = '";
```

Comment:
That's correct. There's an active column on the record type. I'm just referring to the 'ALL' scenario where every record type is pulled: those should only be the active record types.